composer.json

```diff
@@ -1,10 +1,25 @@
 {
-    "require-dev": {
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
-        "wimg/php-compatibility": "dev-master",
-        "wp-coding-standards/wpcs": "dev-develop"
-    },
-    "scripts": {
-        "lint": "phpcs"
-    }
+  "name": "wordpress/gutenberg",
+  "type": "wordpress-plugin",
+  "license": "GPL-2.0",
+  "description": "Prototyping since 1440. Development hub for the editor focus in core.",
+  "homepage": "https://wordpress.github.io/gutenberg/",
+  "keywords": [
+    "gutenberg", "wordpress", "editor", "wp", "react", "javascript"
+  ],
+  "support": {
+    "issues": "https://github.com/WordPress/gutenberg/issues"
+  },
+  "require-dev": {
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
```

Comment:
The early `0.4.*` releases had a couple of bugs. The constraint should allow the most recent anyway, but worth specifying `^0.4.3` just to be sure?